### ISSUE TITLE
更换自己数据集遇见的问题

### ISSUE THREAD

**Issue by warmful-sun** [thuml/Time-Series-Library#433]
尊敬的作者您们好：
我在跑自己的数据集时，数据效果很差，如下：
>>>>>>>testing : long_term_forecast_May_96_12_TimesNet_custom_ftMS_sl96_ll48_pl12_dm64_nh8_el2_dl1_df64_expand2_dc4_fc3_ebtimeF_dtTrue_test_0<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
test 4885
test shape: (305, 16, 12, 1) (305, 16, 12, 1)
test shape: (4880, 12, 1) (4880, 12, 1)
mse:0.9625673294067383, mae:0.6606048345565796, dtw:-999

我是利用长时间序列预测，设置的参数如下：
False
Args in experiment:
Basic Config
  Task Name:          long_term_forecast  Is Training:        1                   
  Model ID:           May_96_12           Model:              TimesNet            

Data Loader
  Data:               custom              Root Path:          ./datasets/WQX/     
  Data Path:          Data_May.csv        Features:           MS                  
  Target:             OT                  Freq:               t                   
  Checkpoints:        ./checkpoints/      

Forecasting Task
  Seq Len:            96                  Label Len:          48                  
  Pred Len:           12                  Seasonal Patterns:  Monthly             
  Inverse:            0                   

Model Parameters
  Top k:              5                   Num Kernels:        6                   
  Enc In:             6                   Dec In:             6                   
  C Out:              6                   d model:            64                  
  n heads:            8                   e layers:           2                   
  d layers:           1                   d FF:               64                  
  Moving Avg:         25                  Factor:             3                   
  Distil:             1                   Dropout:            0.1                 
  Embed:              timeF               Activation:         gelu                
  Output Attention:   0                   

Run Parameters
  Num Workers:        10                  Itr:                1                   
  Train Epochs:       20                  Batch Size:         16                  
  Patience:           5                   Learning Rate:      0.0001              
  Des:                test                Loss:               MSE                 
  Lradj:              type1               Use Amp:            0                   

GPU
  Use GPU:            0                   GPU:                0                   
  Use Multi GPU:      0                   Devices:            0,1,2,3             

De-stationary Projector Params
  P Hidden Dims:      128, 128            P Hidden Layers:    2     
  
这些参数设置，是存在什么问题吗？请大家能够帮忙给出一些提示，感谢！

**Comment by wuhaixu2016** [comment]
您好，我们主要关注算法库开发，不对接具体任务的应用效果问题。
一般来说，需要确认一下您的训练数据是否充足；可以根据一些预测样例，判断是否是任务本身难度太大；也可以跑一下算法库之中其他的模型，看看是否是模型本身不适合该问题。
